documentation/specs/src/economics/public-goods-funding.md

```diff
@@ -38,8 +28,243 @@ Namada groups public goods into four categories, with earmarked pools of funding
 
 ### Funding amounts
 
-In Namada, 10% inflation per annum of the NAM token is directed to this public goods mechanism, 5% to continuous funding and 5% to retroactive funding. This is a genesis default and can be altered by governance.
+In Namada, up to 10% inflation per annum of the NAM token is directed to this public goods mechanism. The division of these funds is entirely the discretion of the elected PGF council.
 
 Namada encourages the public goods council to adopt a default social consensus of an equal split between categories, meaning 1.25% per annum inflation for each category (e.g. 1.25% for technical research continuous funding, 1.25% for technical research retroactive PGF). If no qualified recipients are available, funds may be redirected or burnt.
 
-Namada also pays the public goods council members themselves (in total) a default of 0.1% inflation per annum.
\ No newline at end of file
+Namada also encourages the PGF council to allocate the PGF council members up to 5% of these funds (0.5% inflation) as a payment for value added from the council itself.
+
+## Voting for the Council
+
+### Constructing the council
+All valid PGF councils will be established multisignature account addresses. These must be created by the intdended parties that wish to create a council. The council will therefore have the discretion to decide what threshold will be required for their multisig (i.e the "k" in the "k out of n").
+
+A new multisignature address will be constructed through the CLI with the following command
```

Comment:
This is great but it should be in the docs not the specs (we don't need command examples in the specs)

Reply:
:+1: